src/main/java/com/entropyteam/entropay/employees/jobs/AssignmentJob.java

```diff
@@ -0,0 +1,44 @@
+package com.entropyteam.entropay.employees.jobs;
+
+import java.time.LocalDate;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import com.entropyteam.entropay.employees.repositories.AssignmentRepository;
+
+
+@Component
+public class AssignmentJob {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+    private final AssignmentRepository assignmentRepository;
+
+    @Autowired
+    public AssignmentJob(AssignmentRepository assignmentRepository) {
+        this.assignmentRepository = assignmentRepository;
+    }
+
+    //Job to execute every day at 9:00 AM
+    @Scheduled(cron = "0 5 9 * * ?")
```

Comment:
Should it be  @Scheduled(cron = "0 5 9 * * ?")?

Reply:
Yes, i don't want to run 2 job at the same time